apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/eobByPatientIdAll.json

```diff
@@ -91,6 +91,9 @@
           "system" : "https://bluebutton.cms.gov/resources/variables/fi_orig_clm_cntl_num",
           "value" : "373273882012"
         }
+      }, {
+        "url" : "https://bluebutton.cms.gov/resources/variables/fi_clm_proc_dt",
+        "valueDate" : "2011-02-18"
```

Comment:
Can @williamharty confirm this looks correct? I think this is our first valueDate so just to double check this looks as expected.

Reply:
I did look at this specifically as I think you may be correct that this is our first extension with a valueDate data type.  I did confirm this is correct.